dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/net/info/BroadcastReceiverNetworkInfoProviderTest.kt

```diff
@@ -247,7 +248,7 @@ internal class BroadcastReceiverNetworkInfoProviderTest {
             .hasCellularTechnology(mobileSubtypeNames[subtype])
     }
 
-    @Test
+    @RepeatedTest(4)
```

Comment:
why repeat it 4 times ? Is this number following some sort of best practices ?

Reply:
no, just some reasonable number to cover more branches for the possible test inputs (`forge.anElementFrom`), so that we have more consistency between test suite runs in terms of the branch checked